koala/ast/graph.py

```diff
@@ -745,8 +748,11 @@ def gen_graph(self):
             deps = uniqueify(deps)
 
             for dep in deps:
+                if dep in self.named_ranges:
```

Comment:
Actually named_ranges are added to the graph, it will works if the target cell contains a value or if it’s in the first round list (seeds).
It didn’t work for the NPV case, let’s try this, we’ll rollback if it’s not necessary.

> Le 3 mai 2016 à 08:52, iOiurson <notifications@github.com> a écrit :
> 
> In koala/ast/graph.py <https://github.com/anthill/koala/pull/10#discussion_r61843526>:
> 
> > @@ -745,8 +748,11 @@ def gen_graph(self):
> >              deps = uniqueify(deps)
> >  
> >              for dep in deps:
> > +                if dep in self.named_ranges:
> Could you verify that named range were actually not added to the graph ?
> 
> I think maybe this https://github.com/SergeBredin/koala/blob/master/koala/ast/graph.py#L746 <https://github.com/SergeBredin/koala/blob/master/koala/ast/graph.py#L746> does the trick.
> 
> I'm saying that because in my basic_evaluation.py example, i have no problem using named ranges and finding the correct result.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/anthill/koala/pull/10/files/cd9838d91fc5cff4eeb46675cf492a57b46bd9c6#r61843526>